### PR TITLE
Add instructions on how to run dev builds with a profile

### LINKED ISSUE #274: Document instructions on how to run dev builds with a pre-created profile
discussed with @linuxwolf: adding instructions on how to create a dummy profile with data and set up jpm to point to that profile for quick testing

### PR DESCRIPTION
**Pull request by devinreams**
Closes #274 by adding instructions on how to create a new profile, add it to your jpm config, and run it so it persists between runs. This way testers and developers can set up dummy data locally instead of having to do it every single time.

I've been doing this for a day or so. I know @linuxwolf said he was doing his own `cp -R` shenanigans to achieve roughly the same effect.

Either way, following these instructions mean my test profile opens with ~dozens of entries pre-loaded.

I know @jimporter also wanted to do #170 (a build config) but I wanted to quickly document this approach for those that want/need a good solution sooner.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -42,13 +42,35 @@ To **run the extension in a Firefox Nightly** browser, you can run:
...
+
+To **run the extension with a profile that persists** between runs, you can
+create a new profile by browsing to `about:profiles`.
+
+Once you have a new profile created (no matter the location), you can tell jpm
+(via npm) to run using that profile _and_ not to copy the profile temporarily
+so that any changes (e.g. adding new entries) will be saved and persist:
+
+```sh
+npm run-script run -- -p /path/to/profile --no-copy
```
Comment (pdehaan) [inline docs/install.md]: Not sure if we want to shorten this to `npm run run` (to be consistent with line 76 below).

File: docs/install.md [hunk 2]
```diff
@@ -42,13 +42,35 @@ To **run the extension in a Firefox Nightly** browser, you can run:
...
+between runs. This means the data will be lost every time.
+
+## Running the extension with a persistent profile
+
+To **run the extension with a profile that persists** between runs, you can
+create a new profile by browsing to `about:profiles`.
+
+Once you have a new profile created (no matter the location), you can tell jpm
+(via npm) to run using that profile _and_ not to copy the profile temporarily
+so that any changes (e.g. adding new entries) will be saved and persist:
```
Comment (pdehaan) [inline docs/install.md]: "persist" or "persisted"?

File: docs/install.md (hunk 1 above)
Comment (devinreams) [reply docs/install.md]: Ah yes, I think we can use the "run" short-hand a bit more consistently.

File: docs/install.md (hunk 2 above)
Comment (devinreams) [reply docs/install.md]: Either way, redundant. Removed 😏

File: docs/install.md [hunk 3]
```diff
@@ -39,16 +39,38 @@ you'll need to flip the `extensions.legacy.enabled` preference, too.
...
+
+To **run the extension with a profile that persists** between runs, you can
+create a new profile by browsing to `about:profiles`.
+
+Once you have a new profile created (no matter the location), you can tell jpm
+(via npm) to run using that profile _and_ not to copy the profile temporarily
+so that any changes (e.g. adding new entries) will be saved:
+
+```sh
+npm run run -- -p /path/to/profile --no-copy
```
Comment (linuxwolf) [inline docs/install.md]: `-p profile-name` is just as acceptable (and arguably easier to remembertype) as  `/path/to/profile`.  The latter is necessary if the profile isn't somewhere Firefox already knows to find it

File: docs/install.md [hunk 4]
```diff
@@ -39,16 +39,38 @@ you'll need to flip the `extensions.legacy.enabled` preference, too.
...
+Now when you run using this new profile, any data or settings you make to the
+browser itself or in Lockbox will be available for future runs.
+
+## Setting npm run flags 
+
 To specify flags for `run` to use regularly, use `npm config set jpm_runflags`:
 
 ```sh
-npm config set jpm_runflags="-b nightly"
+npm config set jpm_runflags="-b nightly -p /path/to/profile --no-copy"
```
Comment (linuxwolf) [inline docs/install.md]: \[ see above ]

**Comment by devinreams** [comment]
@m8ttyB @linuxwolf Thanks for double checking my quick doc update. I'll go ahead and merge but if anyone wants to make more tweaks, please feel free. 😁
